Acro-Geniu .py

```
@@ -0,0 +1,30 @@
+import pandas as pd
```

Ruff (0.8.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)